editor/editor_vcs_interface.cpp

```diff
@@ -152,6 +260,190 @@ String EditorVCSInterface::get_vcs_name() {
 	return call("_get_vcs_name");
 }
 
+void EditorVCSInterface::set_up_credentials(String p_username, String p_password) {
+	call("_set_up_credentials", p_username, p_password);
+}
+
+Dictionary EditorVCSInterface::create_diff_line(int new_line_no, int old_line_no, String content, String status) {
```

Comment:
A small number of functions below this are not using the `p_argument_name` convention. Just go over the PR diff once to see if there are any others as well. I don't feel this should affect the operation of the plugin but it wouldn't hurt to check.